### PR TITLE
Fix @Ignored kinesis integration test to not test service behavior.

### PR DESCRIPTION
**Pull request by millems**
It seems like this test started to fail because Kinesis changed the way merging shards worked - with records not being lost in merging. That makes sense, but we relied on the old behavior. Instead of just updating the assertions, updated the test to test less of the service's behavior without testing much less of the SDK behavior.

### CODE REVIEW

File: services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java [hunk 1]
```diff
@@ -133,13 +124,9 @@ public void testKinesisOperations() throws Exception {
...
             Shard shard = shards.get(0);
 
-            // Just to be really sure in case of eventual consistency...
-            Thread.sleep(5000);
-
-            testPuts(streamName, shard);
-
-            // Wait a bit to make sure the records propagate.
-            Thread.sleep(5000);
+            System.out.println("Putting two records...");
```
Comment (dagnir) [inline services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java:127]: nit: remove println

**Review by dagnir — APPROVED** [review]

File: services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java (hunk 1 above)
Comment (millems) [reply services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java:127]: There's a lot of printlns. Should I remove them all?

Comment (dagnir) [reply services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java]: Ah that would be great!

Comment (millems) [reply services/kinesis/src/it/java/software/amazon/awssdk/services/kinesis/KinesisIntegrationTests.java]: Done

**Review by dagnir — APPROVED** [review]

**Pull request merged**